tests/main/unity/task.yaml

```diff
@@ -3,7 +3,7 @@ summary: Check that a unity snap can start and its window is shown
 environment:
     DISPLAY: ":99.0"
 
-systems: [ubuntu-16.04-64-grub,ubuntu-16.04]
+systems: [ubuntu-16.04-64-grub]
```

Comment:
These will need to be updated too.. core to append -64, and the pc ones to drop -grub.